webapp/client/src/pages/RevocationPage.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import FormFieldIdNr from "../components/FormFieldIdNr";
+import FormFieldDate from "../components/FormFieldDate";
+import FormHeader from "../components/FormHeader";
+import FormRowCentered from "../components/FormRowCentered";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import { fieldPropType } from "../lib/propTypes";
+
+export default function RevocationPage({ stepHeader, form, fields }) {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <StepHeaderButtons />
+      <FormHeader {...stepHeader} />
+      <StepForm {...form}>
+        <FormRowCentered>
+          <FormFieldIdNr
```

Comment:
I know that this is not that important and it was part of the old version as well but why don't we use the detail here but use it on all the other pages? Maybe, we could talk 1 min. to Tine/Nadine and then add. Or we leave as is and file a "bug ticket" to prioritize it?

Reply:
I think I actually know why. If you landed here, you already at least registered so you know what you're IdNr ist. But maybe that's wrong and I'm gonna quickly ask Nadine.

Reply:
I asked her, she said, "Yes please thank you" - so another thanks to you^^